msg/follow_target_status.msg

```diff
@@ -1,5 +1,11 @@
-uint64 timestamp             # time since system start (microseconds)
+uint64 timestamp             # [microseconds] time since system start
+
+float32[3] pos_est_filtered  # [m/s] Filtered target position used for vehicle's setpoint generation
+float32[3] vel_est_filtered  # [m/s] Filtered Target velocity used for vehicle's setpoint generation
+
+float32 tracked_target_orientation # [rad] Tracked target orientation, which can be rate constrained
```

Comment:
Terminology wise, I'm still confused but yes it's global (and local) NED frame based heading! (North is 0 orientation angle)

Reply:
call it `tracked_target_heading` maybe?

Reply:
`heading` is the angle between North and the `air velocity`
`yaw` is the angle between North and the `x-body-axis`
`course` is the angle between North and the `ground velocity`

I originally proposed `orientation` here - because I didnt know if we would switch back and forth between `course` based operations (as it is now) and `yaw` based operations, as could be if we had feedback from the rover, or car, or whatever. `orientation` is then more general. Also the definition is even now a bit ambiguous in our current implementation, because course is undefined at zero speed. If you were to change it, I would only recommend the word `course`, as that's how it's (mostly) implemented now, and I'm on a crusade to lock down consistent aerospace namings :)

but otherwise -- i think if you define it well in the message definition, can keep it as is.

Reply:
I will opt for course!